ISICkonto/AppDelegate.swift

```diff
@@ -7,40 +7,74 @@
 //
 
 import UIKit
+import SVProgressHUD
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
+    
     var window: UIWindow?
-
-
+    var appFlowController: AppFlowController?
+    
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        
+        setUpGlobalAppearance()
+        
+        window = UIWindow(frame: UIScreen.main.bounds)
+        
+        if let window = window {
+            window.rootViewController = UINavigationController()
+            appFlowController = AppFlowController(with: window.rootViewController as! UINavigationController)
```

Comment:
❔ nebylo by lepší vytvořit si lokální proměnnou `let rootViewController = UINavigationController()`, kterou potom předat to `window.rootViewController` a i do `AppFlowController(with: rootViewController)`? Vyhneš se tak tomu force castu z `window.rootViewController` na `UINavigationController`

Reply:
jasne, makes sense, to me nenapadlo